src/bootstrap.ts

```diff
@@ -123,22 +124,24 @@ const setupRxNormConnection = (conf: RxNormDbConfig) => {
     timeout: conf.timeout,
   });
 
-  pool.on('error', err => setStatus('rxNormDb', { status: Status.ERROR }));
   pool.on('connection', () => setStatus('rxNormDb', { status: Status.OK }));
+
+  setStatus('rxNormDb', { status: Status.OK });
```

Comment:
won't this always be status `OK` now?? I don't see anywhere it would be set to error.

Reply:
yep, it was only checking on the beginning I tried set interval and ping the db but that didn't work.. that check missed up with the tests somehow. so for now I just want to get done with it, not urgent thing

Reply:
cool as long as we don't forget it.